src/components/input.js

```diff
@@ -264,7 +273,7 @@ const withAutocomplete = WrappedComponent => ({
         h(WrappedComponent, getInputProps({
           style: inputIcon ? { ...style, paddingLeft: '3rem' } : style,
           type: 'search',
-          onKeyDown: e => {
+          onKeyUp: e => {
```

Comment:
This is a lifecycle difference, that caused the backspace bug - KeyUp runs after the value has updated, KeyDown runs before the input value updates